src/platforms/python/common/integrations/aws-lambda/manual-layer/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: AWS Lambda Layer
+description: "Learn how to manually set up Sentry with a Layer"
+sidebar_order: 1000
+---
+
+You can add Sentry to your AWS Lambda function by adding the Sentry AWS Lambda Layer to your function.
+
+## Install the layer
+
+Navigate to your Lambda function:
+
+![](lambda-function-detail.png)
```

Comment:
Is there any way to make the text in these screenshots a bit bigger? It's really hard to read.